src/main/resources/locale_fr_FR.properties

```
@@ -0,0 +1,207 @@
+command.scan.scheduling=Planifie ton analyse. Le temps nécessaire estimé jusqu'à l'achèvement est de : %DURATION%.
+command.scan.sub.cancel=Annule un scan en cours.
+command.scan.sub.cancel.noTask=Aucune tâche n'est en cours pour cette guilde.
+command.setup.description=Exécute une configuration de base du robot.
+command.setup.dialog.channels=Veuillez saisir tous les canaux où les utilisateurs sont autorisés à recevoir de la réputation.
+command.setup.dialog.channels.allChannel=Tous les canaux
+command.setup.dialog.continueToProceed=Veuillez appuyer sur "$word.continue$" pour continuer.
+command.setup.dialog.loadDefaults=Charge les mots de remerciement par défaut pour tes langues.
+command.setup.dialog.managerRole=Veuillez écrire un rôle qui doit être autorisé à modifier les paramètres du robot.
+command.setup.dialog.roles=Veuillez définir des rôles de réputation.\nCes rôles seront attribués à un utilisateur lorsqu'il atteindra le niveau de réputation spécifié.\nDéfinis-les comme suit : <$words.role$> <$words.reputation$>.\nLorsque tu as défini tous tes rôles, appuie sur le bouton.
+command.setup.dialog.rolesFormat=Veuillez les définir comme <$words.role$> <$words.reputation$>.
+command.setup.dialog.selectLanguage=Veuillez sélectionner votre langue.
+command.setup.dialog.setupComplete=Installation terminée !
+command.setup.dialog.starting=Démarre l'installation.
+command.setup.dialog.welcome=Bienvenue dans la configuration du robot de réputation.
+command.thankwords.description=Configurer Thankwords
+command.thankwords.error.patternNotFound=Le modèle n'existe pas.
+command.thankwords.sub.add=Ajouter un autre motif. Les motifs sont interprétés comme des Regex. Un motif ne peut pas contenir d'espaces.
+command.thankwords.sub.add.added=Ajouté le motif %REGEX%.
+command.thankwords.sub.check=Vérifie si un message donne une réputation.
+command.thankwords.sub.check.confidence=Score de confiance : %SCORE%.
+command.thankwords.sub.check.match.answer=Mise en correspondance des réponses
+command.thankwords.sub.check.match.fuzzy=Correspondance floue
+command.thankwords.sub.check.match.mention=Mention Match
+command.thankwords.sub.check.match.noMatch=No Match
+command.thankwords.sub.check.notConfident=Le score de confiance est trop faible. Une réputation ne sera pas donnée.
+command.thankwords.sub.check.reference=Message de référence : [Sauter](%URL%)
+command.thankwords.sub.check.result=Donateur : %DONATOR%\nReceveur : %RECEIVER%.
+command.thankwords.sub.list=Liste de tous les motifs
+command.thankwords.sub.list.list=Les motifs suivants sont actifs :
+command.thankwords.sub.loadDefault=Charge les mots de remerciement par défaut pour une langue.
+command.thankwords.sub.loadDefault.added=Ajoute les mots de remerciement suivants :
+command.thankwords.sub.loadDefault.available=Les langues disponibles :
+command.thankwords.sub.remove=Supprimer un motif
+command.thankwords.sub.remove.removed=Removed pattern %PATTERN%.
```

Looks untranslated

DeepL suggestion:
```suggestion
command.thankwords.sub.remove.removed=Modèle supprimé %PATTERN%.
```